workspace.json

```diff
@@ -319,6 +331,18 @@
             "buildTarget": "api:build"
           }
         },
+        "build-schema": {
+          "builder": "@nrwl/workspace:run-commands",
+          "options": {
+            "command": "yarn ts-node -P apps/api/tsconfig.app.json apps/api/src/buildSchema.ts"
+          }
+        },
+        "init-schema": {
+          "builder": "@nrwl/workspace:run-commands",
+          "options": {
+            "command": "yarn nx run api:codegen"
+          }
+        },
```

Comment:
What's the difference between `build-schema`, `init-schema` and `codegen` (for developer workflow)?

Reply:
- `build-schema` is used on the graphql module where autoSchemaFile is defined to generate the `api.graphql` file
- `codegen` is used in projects where a codegen.yml file exists
- `init-schema` is the common script used with the nx affected. For nestjs project it will run in general `build-schema`, for web projects it will run `codegen` scripts and for open api projects it will run `buildOpenApi`.

Reply:
Actually, `build-schema` is only used once for the API because I run it before all the affected `init-schema`. 

Otherwise, `init-schema` run either or together `buildSchema.ts`/`codegen`/`buildOpenApi.ts` if the project is using Nest modules, web stack or open api.